app/exerlog/lib/src/widgets/gradient_button.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+class RaisedGradientButton extends StatefulWidget {
+  RaisedGradientButton({
+    required this.onPressed,
+    required this.child,
+    this.width = double.infinity,
+    this.height = 50.0,
+    this.radius = 30,
+    this.gradient,
+  });
+
+  final FutureOr Function() onPressed;
+  final Widget child;
+  final double width;
+  final double height;
+  final double radius;
+  final Gradient? gradient;
+
+  @override
+  _RaisedGradientButtonState createState() => _RaisedGradientButtonState();
+}
+
+class _RaisedGradientButtonState extends State<RaisedGradientButton> {
+  bool _isLoading = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: widget.height,
+      width: widget.width,
+      child: DecoratedBox(
+        decoration: BoxDecoration(
+          gradient: widget.gradient ??
+              LinearGradient(
+                colors: <Color>[
+                  Color(0xFF34D1C2),

```

Comment:
Don't know if it's in the scope of this PR, but we should probably add a theme file of sorts to store all the colors that are being used in the app, instead of redefining them everytime we use a color.